lib/models/tree-node.model.ts

```diff
@@ -151,7 +152,7 @@ export class TreeNode implements ITreeNode {
     });
   }
 
-  allowDrop(element) {
+  allowDropTemplate(element) {
```

Comment:
I can't make it private as templates cannot call private methods. It works with the JIT because that one generates JavaScript which does not have the private/public distinction but `ngc` (the AoT compiler) generates TypeScript which throws an error when templates try to access private instance members.

As to publishing the compiled files, I am not sure but if @vsavkin does it, it's probably right 😄  I've seen it in other components as well.